locales/fr.json

```diff
@@ -145,7 +145,7 @@
     },
     "search": "rechercher",
     "Log out": "Se déconnecter",
-    "Released under the AGPLv3 by Omar Roth.": "Publié sous licence AGPLv3 par Omar Roth.",
+    "Released under the AGPLv3 on Github.": "Publié sous licence AGPLv3 par Omar Roth.",
```

Comment:
French translation still mentions Omar Roth

Reply:
Oops! I pasted the translated string but I forgot to edit it, fixed.